rsrc/dialogs/edit-scenario-details.xml

```diff
@@ -2,30 +2,39 @@
 <!-- NOTE: This file should be updated to use relative positioning the next time it changes. -->
 <?xml-stylesheet href="dialog.xsl" type="text/xsl"?>
 <dialog defbtn='okay'>
-	<field name='ver1' type='uint' top='80' left='181' width='43' height='16'/>
-	<field name='ver2' type='uint' top='80' left='234' width='43' height='16'/>
-	<field name='ver3' type='uint' top='80' left='287' width='43' height='16'/>
-	<field name='who1' top='104' left='181' width='243' height='47'/>
-	<field name='who2' top='159' left='181' width='243' height='47'/>
-	<field name='contact' top='214' left='181' width='243' height='65'/>
-	<button name='okay' type='regular' top='528' left='395'>OK</button>
 	<pict type='dlog' num='16' top='8' left='8'/>
+	<text size='large' top='6' left='50' width='256' height='17'>Scenario Details</text>
+
 	<text top='23' left='50' width='372' height='53'>
 		This is where you can define the various pieces of information the user will see when deciding whether or not to play your scenario.
 		The meanings of all these fields are given in the documentation.
 	</text>
-	<text size='large' top='6' left='50' width='256' height='17'>Scenario Details</text>
-	<text top='81' left='50' width='120' height='14'>Version number:</text>
-	<text top='102' left='50' width='120' height='14'>Credits, Part 1:</text>
-	<text top='150' left='50' width='120' height='14'>Credits, Part 2:</text>
-	<text top='213' left='50' width='120' height='14'>Contact Information:</text>
+
+	<text name='title-text' top='65' left='50' width='120' height='14'>Title:</text>
```

Comment:
I think the ID (`name`) for these various `<text>` elements should end in something other than `-text` – to me, that implies it should hold the actual title, not a label for the title. `-label` is the most obvious alternative. This applies to all the `<text>` elements that gained a new ID.